comma/individual.py

```diff
@@ -87,6 +87,26 @@ def take_actions(self, actions: pd.Series):
             results.append(result)
         self._status.loc[self.id] = results
        
+    @staticmethod    
+    def data_sampling_ipf(size: int, dir_params: str):
+        """Sample data items from ipf weights.
+
+        Args:
+            size (int): size of data sample.
+            dir_params (str): path to paramters folder.
+        """
+        fpath_weights = os.path.join(dir_params, PARAMS_IPF_WEIGHTS) 
+        assert os.path.isfile(fpath_weights)
```

Comment:
Since this is not crucial, and it's @jiqicn's code, I think should be his decision how/if to change this. Again, by working with different engineers, I hardly find a consensus about this issue, i.e., asserting file paths before loading it. I agree with you that it seems redundant, perhaps Ji has a different opinion about this.